Assets/Scripts/DataModels/Pilots/PilotAccessors.cs

```diff
@@ -38,7 +39,10 @@ public bool IsAssignedToShip
         get => saveData.isAssignedToShip; set { saveData.isAssignedToShip = value; }
     }
 
-    public Ship Ship { get => ship; }
+    public Ship Ship 
+    {
+        get => ShipsManager.Instance.Ships.FirstOrDefault(s => s.Pilot == this); 

```

Comment:
We don't want the Pilot model having a Ship on it while the Ship has a Pilot on it (circular). So, the Ship field has been deleted on the Pilot and replaced with this. Probably a better way.

Reply:
i couldn't figure out a better way than the circular reference :(